src/core/LabeledInput/LabeledInput.tsx

```diff
@@ -102,11 +117,13 @@ export const LabeledInput = React.forwardRef<
         ref={ref}
         {...rest}
       />
-      {(message || icon) && (
-        <div className='iui-message'>
-          {icon}
-          {displayStyle === 'default' && message}
-        </div>
+      {icon &&
+        React.cloneElement(icon, {
```

Comment:
Everywhere it is the same. Maybe we need `Label` type component? At first it can even be inner, later we could make it public.

Reply:
`Label` or `InputIcon` ? dont understand which part you mean

Reply:
Label with icon and message.

Reply:
not sure, how label with icon and message could look like..?

Reply:
Just HOC/wrapper component, you won't use it alone.